src/io/keyboard.js

```diff
@@ -1,11 +1,16 @@
 var Cast = require('../util/cast');
 
-function Keyboard () {
+function Keyboard (rt) {
     /**
      * List of currently pressed keys.
      * @type{Array.<number>}
      */
     this._keysPressed = [];
+    /**
+     * For activate hat
+     * @type{Runtime}
```

Comment:
`@type{!Runtime}`. Runtime can't be null - if it is, it would throw errors when you try to activate hats.